docs/mapview.md

```diff
@@ -64,6 +64,8 @@ To access event data, you will need to use `e.nativeEvent`. For example, `onPres
 |---|---|---|
 | `animateToRegion` | `region: Region`, `duration: Number` |
 | `animateToCoordinate` | `coordinate: LatLng`, `duration: Number` |
+| `animateToBearing` | `bearing: Number`, `duration: Number` |
+| `animateToViewingAngle` | `angle: Number`, `duration: Number` |
```

Comment:
@christopherdro I think MapKit also allows you to set the bearing in degress, seems that they only call it differently (heading):
```
var heading: CLLocationDirection
The heading of the camera (measured in degrees) relative to true north.
```
I feel that bearing (https://en.wikipedia.org/wiki/Bearing_(navigation)) is a more proper name. 
For the Viewing Angle of the camera I am not so sure if ViewingAngle is better than pitch (how MapKit on IOS call it):
```
var pitch: CGFloat
The viewing angle of the camera, measured in degrees.
```

I think is possible to add the support for MapKit (IOS provided) as I did for GoogleMaps, the only thing is I want to know your position regarding naming? Should we follow GoogleMap or MapKit name convention?